runner/src/mill/runner/MillMain.scala

```diff
@@ -79,6 +79,40 @@ object MillMain {
 
   lazy val maybeScalaCompilerWorker = ScalaCompilerWorker.bootstrapWorker()
 
+  private def withStreams[T](
+      bspMode: Boolean,
+      streams0: SystemStreams
```

Comment:
Can you give `streams0` a more descriptive name? Since it's not just `streams`, it must be something special.

Reply:
(renamed it back to `streams`)